extensions/ui/org.eclipse.smarthome.ui.basic/src/main/java/org/eclipse/smarthome/ui/basic/internal/servlet/WebAppServlet.java

```diff
@@ -144,7 +144,11 @@ public void service(ServletRequest req, ServletResponse res) throws ServletExcep
             if (widgetId == null || widgetId.isEmpty() || widgetId.equals(sitemapName)) {
                 // we are at the homepage, so we render the children of the sitemap root node
                 if (subscriptionId != null) {
-                    subscriptions.setPageId(subscriptionId, sitemap.getName(), sitemapName);
+                    if (subscriptions.exists(subscriptionId)) {
+                        subscriptions.setPageId(subscriptionId, sitemap.getName(), sitemapName);
+                    } else {
+                        logger.warn("Basic UI requested a non-existing event subscription id ({})", subscriptionId);

```

Comment:
Note that I only copied&pasted this from https://github.com/kaikreuzer/smarthome/blob/a2a238c39b9128a41041b67f6ac2356bdbaabc58/extensions/ui/org.eclipse.smarthome.ui.basic/src/main/java/org/eclipse/smarthome/ui/basic/internal/servlet/WebAppServlet.java#L158-L162, there the very same is done. But yes, I can reduce both places to debug logging.